src/Microsoft.DotNet.Build.Tasks/GenerateAssemblyList.cs

```diff
@@ -31,12 +30,6 @@ public override bool Execute()
                 while ((line = listReader.ReadLine()) != null)
                 {
                     var fileName = Path.GetFileName(line);
-                    if (!processedFileNames.Add(fileName))
```

Comment:
Noted. This Assembly List is being phased out, as more teams start using the universal test runner.